orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -2043,7 +2049,12 @@ TEST(GradientCheckerTest, GatherNDGrad_unique_float_data) {
 }
 
 TEST(GradientCheckerTest, LayerNormGrad) {
```

Comment:
These tests are enabled only for `USE_CUDA`, should we also include them for ROCm EP?

Reply:
We need to evaluate the time cost when enable them to ROCm, not in this PR.